css/style.css

```diff
@@ -0,0 +1,3958 @@
+}
+
+.footer-nav-link:hover,
+.footer-nav-link:active,
+.footer-nav-link:focus {
+  color: var(--primary-color);
+  border-bottom: 1px solid var(--primary-color);
+}
+
+.newsletter-form {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.newsletter-label {
+  max-width: 250px;
+  width: 100%;
+  font-family: var(--font-header);
+  font-weight: var(--font-bold);
+  font-size: 24px;
+  line-height: 1.3;
+  padding-right: 20px;
+}
+
+.newsletter-input {
+  margin-left: auto;
+  padding: 20px;
+  max-width: 320px;
+  width: 100%;
+  -ms-flex-item-align: start;
+  -ms-grid-row-align: start;
+  align-self: start;
+  color: var(--primary-color);
+  background-color: var(--secondary-color);
+  border-radius: var(--border-radius-s);
+  border: 1px solid var(--opacity-15);
+  -webkit-transition: border var(--trasition-effect);
+  -o-transition: border var(--trasition-effect);
+  transition: border var(--trasition-effect);
+}
+
+.newsletter-input::-webkit-input-placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.newsletter-input::-moz-placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.newsletter-input:-ms-input-placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.newsletter-input::-ms-input-placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.newsletter-input::placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.newsletter-input::-webkit-input-placeholder::first-letter {
+  text-transform: capitalize;
+}
+
+.newsletter-input::-moz-placeholder::first-letter {
+  text-transform: capitalize;
+}
+
+.newsletter-input:-ms-input-placeholder::first-letter {
+  text-transform: capitalize;
+}
+
+.newsletter-input::-ms-input-placeholder::first-letter {
+  text-transform: capitalize;
+}
+
+.newsletter-input::placeholder::first-letter {
+  text-transform: capitalize;
+}
+
+.newsletter-input:-webkit-autofill,
+.newsletter-input:-webkit-autofill:hover,
+.newsletter-input:-webkit-autofill:focus {
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: var(--opacity-70);
+  -webkit-box-shadow: 0 0 0 50px var(--secondary-color) inset;
+}
+
+.newsletter-input:hover,
+.newsletter-input:focus {
+  outline: none;
+  border: 1px solid var(--accent-color);
```

Comment:
You can use `boder-color` instead - only this property changed

Reply:
#Fixed